src/uwtools/tests/config/formats/test_fieldtable.py

```diff
@@ -1,39 +1,48 @@
-# pylint: disable=missing-function-docstring
+# pylint: disable=missing-function-docstring,redefined-outer-name
```

Comment:
I wouldn't mind having an in-person chat about this annoying linter thing some time.